.github/workflows/gradle.yml

```diff
@@ -0,0 +1,21 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Grant rights build.gradle
+      run: chmod +x ./backend-api/build.gradle
```

Comment:
are you sure this needs `+x`?

Reply:
no need, just an experiment. It isn't an executable. I will remove it.

Reply:
removed